codalab/rest/bundles.py

```diff
@@ -303,9 +304,10 @@ def _create_bundles():
         # Add as item to worksheet
         if not detached:
             if shadow_parent_uuid is None:
-                local.model.add_worksheet_items(
-                    worksheet_uuid, [worksheet_util.bundle_item(bundle_uuid)], after_sort_key
-                )
+                items = [worksheet_util.bundle_item(bundle_uuid)]
+                if after_image:
```

Comment:
Let's add a comment here to explain what this code is doing

Reply:
solved